client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -63,10 +87,85 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 		setSelectedProtocol( protocol );
 	};
 
```

Comment:
During the review, It was kinda confusing to me to follow the code of `shouldBlockedByPluginUpgradeCheck` `checkMigrationEnabledCallback ` functions. 

Is it possible to move it to custom hooks and simply use it in the `pre-migration` component?
It will be much better to follow as an isolated logic and for maintaining purposes.

Reply:
Thanks for pointing this out @bogiii , you're right. I should have moved this logic away from the `pre-migration` component. Regarding code readability, I've refactored it and made it into a custom hook now, it also splits logic into small function chunks. Could you have another look once you have a moment and let me know if it's clearer/easier to follow?

Reply:
It's much clearer, I appreciate it!

Thanks, @ouikhuan!